identity_core/src/crypto/proof/jcs_ed25519.rs

```diff
@@ -20,11 +20,21 @@ use crate::utils::BaseEncoding;
 // TODO: Marker trait for Ed25519 implementations (?)
 
 /// An implementation of the [JCS Ed25519 Signature 2020][SPEC1] signature suite
-/// for [Linked Data Proofs][SPEC2].
+/// for [Linked Data Proofs][SPEC2] modified to pass the official test vectors.
 ///
 /// Users should use the [`Sign`]/[`Verify`] traits to access
 /// this implementation.
 ///
+/// ## Deviation from the [JCS Ed25519 Signature 2020 specification][SPEC1]
+/// This implementation follows the specification with the single exception that the third step of the [Proof generation algorithm](https://identity.foundation/JcsEd25519Signature2020/#ProofGeneration) is skipped.
+/// In other words this implementation does NOT apply the SHA-256 Hash algorithm before signing.
+///
+/// We have chosen to deviate from the specification on this point in order to pass the [official test vectors](https://github.com/decentralized-identity/JcsEd25519Signature2020/tree/master/signature-suite-impls/test-vectors) as well as achieving compatibility with the reference implementations [for Go](https://github.com/decentralized-identity/JcsEd25519Signature2020/tree/master/signature-suite-impls/golang)
+/// and [for Java](https://github.com/decentralized-identity/JcsEd25519Signature2020/tree/master/signature-suite-impls/java).
```

Comment:
Should we reference the latest commit here specifically or is it fine to just reference their repo?

```suggestion
/// We deviate from the specification to satisfy the [official test vectors](https://github.com/decentralized-identity/JcsEd25519Signature2020/tree/master/signature-suite-impls/test-vectors), as well as to achieve compatibility with the official reference implementations for [Go](https://github.com/decentralized-identity/JcsEd25519Signature2020/tree/master/signature-suite-impls/golang)
/// and [Java](https://github.com/decentralized-identity/JcsEd25519Signature2020/tree/master/signature-suite-impls/java).
```

Reply:
> Should we reference the latest commit here specifically or is it fine to just reference their repo?

If they change their repo then one would expect that would mean fixing their specification? in which case these doc comments will become redundant.

Reply:
I was more thinking they could just move or remove those directories. I'm fine either way.